app/models/github.rb

```diff
@@ -0,0 +1,23 @@
+class Github
+  class Repo
```

Comment:
Really like this as a module & follows the source syntax... great work!

Reply:
Thanks 👍